solidity/contracts/ERC1155MixedFungible.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/utils/Context.sol";
+
+/**
+    @dev Mintable+burnable form of ERC1155 with mixed fungible/non-fungible item support.
+    Based on reference implementation:
+    https://github.com/enjin/erc-1155/blob/master/contracts/ERC1155MixedFungibleMintable.sol
+*/
+contract ERC1155MixedFungible is Context, ERC1155 {
```

Comment:
if we do `contract ERC1155MixedFungible is Context, ERC1155, ERC1155Burnable` then wouldn't the copying of the code can be avoided?

Reply:
The only difference is that the OpenZeppelin [ERC1155Burnable](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC1155/extensions/ERC1155Burnable.sol) doesn't include a `data` argument in its signature. I think we want to allow passing in an arg to associate a token burn with some other piece of data.

Reply:
ah ok, wasn't paying close attention, thought the methods were copied verbatim. that makes sense.